frontend/src/views/RouteFormView.vue

```diff
@@ -114,11 +128,13 @@
 </template>
 
 <script>
+import { MultiSelect } from 'vue-search-select';
```

Comment:
For me it does not work - eslint screams about lack of ".js" extension in import. The solution was to add an empty line after this line 😕

Reply:
It seems to be passing fine on server and on my local machine, but if it helps I am adding an empty line.